helm/operator/templates/minio.min.io_tenants.yaml

```diff
@@ -2467,6 +2478,14 @@ spec:
                         required:
                         - port
                         type: object
+                      sleep:
```

Comment:
@pjuarezd What is `sleep` for?

Reply:
part of pod lifecycle hooks

https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#hook-handler-execution